### PR TITLE
profile customization

### PR DESCRIPTION
**Pull request by motzel**
image

### CODE REVIEW

File: src/components/Player/Profile.svelte [hunk 1]
```diff
@@ -59,19 +63,74 @@
...
+				?.reverse()
+				?.filter(r => r?.length) ?? [];
 	}
 
 	function refreshPinnedScores(pinnedScores) {
 		$pinnedScoresStore = pinnedScores ?? [];
 	}
 
+	function onEnableEditModel() {
+		editModel = editModel = {
```
Comment (NSGolova) [inline src/components/Player/Profile.svelte:84]: Is this some cool trick or just a mistake with two assignments?

File: src/components/Player/Profile.svelte [hunk 2]
```diff
@@ -59,19 +63,74 @@
...
+			data: {
+				name: playerData?.name ?? '',
+				country: playerData?.playerInfo?.countries?.[0]?.country?.toLowerCase() ?? '',
+				avatar: null,
+				patreonMessage: playerData?.playerInfo?.patreonFeatures?.message ?? '',
```
Comment (NSGolova) [inline src/components/Player/Profile.svelte:89]: I moved this to a `profileSettings` as well, please use only them so I can delete `patreonFeatures` in future

File: src/components/Player/AvatarOverlayIcons.svelte [hunk 3]
```diff
@@ -145,7 +123,7 @@
 	<nav class:main={isMain}>
 		{#if loggedInPlayer && !isMain && (showAvatarIcons === 'show' || (showAvatarIcons === 'only-when-needed' && !isFriend))}
 			<Button
-				square={true},
+				square="{true},"
```
[the remark this answers is not in the document]
Comment (motzel) [reply src/components/Player/AvatarOverlayIcons.svelte:126]: I haven't touched it, must be added by prettier. Will fix

File: src/components/Player/Profile.svelte (hunk 1 above)
Comment (motzel) [reply src/components/Player/Profile.svelte:84]: Just a mistake, probably when ctrl+c/v xD

**Pull request merged**